tests/lib/collectors/grr_base.py

```diff
@@ -21,6 +21,10 @@ class MockGRRObject(object):
   """Fake GRR object that will be used in the access forbidden wrapper test"""
   _access_forbidden_counter = 0
   CreateApproval = mock.MagicMock()
+  ClientApproval = mock.MagicMock()
+  ClientApproval.client_id = "abcd"
+  ClientApproval.approval_id = "dcba"
```

Comment:
also add `username` here if you're using `approval.username` before.

Reply:
ack!